src/shareInfo.js

```diff
@@ -5,6 +5,14 @@
  * @classdesc ShareInfo class, stores information regarding a share
  * @param {object} ShareInfo containing information like id, url etc. of the share
  */
+
+const USER_TYPE_USER = 0
+const USER_TYPE_GUEST = 1
+const SHARE_TYPE_USER = 0;
+const SHARE_TYPE_GROUP = 1;
+const SHARE_TYPE_LINK = 3;
+const SHARE_TYPE_GUEST = 4;
```

Comment:
I think it's sufficient to only introduce the constant that you need and link to the documentation for other share types (same link for user type and ping the docs team to add the user type prop to the docs): https://doc.owncloud.com/server/next/developer_manual/core/apis/ocs-share-api.html#response-attributes-2

Reply:
https://github.com/owncloud/docs-server/issues/272#issuecomment-1108501320